pagopa/io-app
# chore(IT Wallet): [SIW-1414] Disable swipe navigation in IT Wallet issuance flows

## Short description
This PR disable swipe back navigation in some IT Wallet issuance screens

## List of changes proposed in this pull request
- Removed swipe back navigation in loading screens
- Removed `ItwIssuanceEidRequestScreen`
- Added `LoadingScreenContent` in credential preview screen

## How to test
Start an eID or credential issuance flow, check that you can't navigate back in loading screen or credential preview screens.
